src/widgets/combobox.ts

```diff
@@ -18,10 +18,10 @@ export class ComboBox extends Widget {
 	}
 
 	public async selectOptionByNumber(i: number): Promise<void> {
-		await (await (await this.elem).$$(`[role='row'][row-index='` + i + `']`))[1].click();
+		await (await this.allByCSS(`[role='row'][row-index='` + i + `']`))[1].click();
 	}
 
     public async selectOptionByText(text: string): Promise<void> {
-		await this.elem.$(`ag-cell-value*=${text}`).click();
+		await (await this.byCSS(`ag-cell-value*=${text}`)).click();
```

Comment:
This is not a CSS selector, but a "by text" search. Thus, although this would work, I would use the corresponding method to avoid not relying on WDIO syntax for this.